src/net_processing.h

```diff
@@ -46,11 +46,15 @@ class PeerManager : public CValidationInterface, public NetEventsInterface
 {
 public:
     struct Options {
-        /** Whether this node is running in -blocksonly mode */
+        //! Whether this node is running in -blocksonly mode
         bool ignore_incoming_txs{DEFAULT_BLOCKSONLY};
+        //! Whether transaction reconciliation protocol is enabled
         bool reconcile_txs{DEFAULT_TXRECONCILIATION_ENABLE};
+        //! Maximum number of orphan transactions kept in memory
         uint32_t max_orphan_txs{DEFAULT_MAX_ORPHAN_TRANSACTIONS};
-        size_t max_extra_txs{DEFAULT_BLOCK_RECONSTRUCTION_EXTRA_TXN};
+        //! Number of orphan+recently-replaced transactions to keep around for block reconstruction
```

Comment:
nit, includes a bit more than replaced and orphans
```suggestion
        //! Number of non-mempool transactions to keep around for block reconstruction. Includes orphan, replaced, and rejected transactions.
```

Reply:
Ah thanks, fixed in aa89e04e07ca9ff51b1d7d310a11821c6ad963cf